packages/logging/charts/logging/charts/logging-operator/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "3.3.0"
+description: A Helm chart to install Banzai Cloud logging-operator
+name: logging-operator
+version: 3.3.0
```

Comment:
should probably add "00" at the end so we can increment chart version without updating app and still maintain the pattern

Reply:
This is sub-chart, is goal here to upstream this all long term ? Then this file wouldn't even exist.